### DISCUSSION TITLE
Migration von IOTA auf IOTA Firefly funktioniert nicht

### DISCUSSION THREAD

**Discussion by flat83** [iotaledger/firefly#7686] [Help & Support]
Migration von IOTA auf IOTA Firefly funktioniert nicht.

In der neuen App für mir immer das angezeigt " Konten werden erstellt"
Was kann ich tun?

**Comment by begonaalvarezd** [comment]
Do you mean legacy migration? If so, the current ability to auto-migrate tokens from the pre-Chrysalis IOTA network via the Firefly wallet have been stopped with the IOTA Stardust upgrade. Once IOTA smart contracts are available on the IOTA network, this feature will be restored and migrations can be processed automatically again
We will provide a migration tool to resume legacy migrations, but its not finished yet

**Reply by flat83** [reply]
Vielen Dank. Dann werde ich warten :-): 
Wo oder wie bekomme ich die Info wenn das Migrationstool fertig ist?

**Reply by begonaalvarezd** [reply]
I presume it will be announced by the IOTA Foundation when ready, in their discord server, social media accounts, blog, etc...
Will close this discussion as the migration tool will not be in Firefly but it will be a different app 🙏🏼

**Reply by flat83** [reply]
Danke
